architecture/port_forwarding.adoc

```diff
@@ -10,48 +10,21 @@
-Support for port forwarding is built into the `kubectl` command:
-
-----
-kubectl port-forward -p <pod> [<local port:]<pod port> [[<local port:]<pod port> ...]
-----
-
-Kubectl listens on each local port specified by the user, forwarding via the link:#protocol[protocol] described below.
-
-Ports may be specified using the following formats:
-
-* `5000` - the client listens on port 5000 locally and forwards to 5000 in the pod
-* `6000:5000` - the client listens on port 6000 locally and forwards to 5000 in the pod
-* `:5000` or `0:5000` - the client selects a free local port and forwards to 5000 in the pod
-
-=== Protocol
-Clients initiate port forwarding to a pod by issuing a request to the
-Kubernetes API server:
-
-----
-/proxy/minions/node123.openshift.com/portForward/myns/mypod
-----
-
-In the above URL:
-
-* `node123.openshift.com` is the node name
-* `myns` is the target pod's namespace
-* `mypod` is the target pod's name
-
-After sending a port forward request to the API server, the client upgrades the connection to one that supports multiplexed streams; the current implementation uses SPDY.
-
-The client creates a stream with the `port` header containing the target port in the pod. All data written to the stream is delivered via the Kubelet to the target pod and port. Similarly, all data sent from the pod for that forwarded connection is delivered back to the same stream in the client.
-
-The client should close all streams, the upgraded connection, and the
-underlying connection when it is finished with the port forwarding request.
+Developers can link:../using_openshift/port_forwarding.html[use the CLI] to port forward to a pod. The CLI listens on each local port specified by the user, forwarding via the link:../using_openshift/port_forwarding.html#protocol[described protocol].
 
 == Server Operation
-The Kubelet handles port forward requests from clients. Upon receiving a request, it upgrades the response and waits for the client to create port forwarding streams. When it receives a new stream, it copies data between the stream and the pod's port.
-
-=== Forwarding to a Pod's Port
-There are a few different options for forwarding to a pod's port. The current implementation invokes `nsenter` directly on the host (node) to enter the pod's network namespace and invoke `socat` to copy data between the stream and the pod's port. Other options include running a "helper" pod that then runs `nsenter` and `socat` so those binaries aren't required to be installed on the host.
+The Kubelet handles port forward requests from clients. Upon receiving a
+request, it upgrades the response and waits for the client to create port
+forwarding streams. When it receives a new stream, it copies data between the
+stream and the pod's port.
+
+Architecturally, there are options for forwarding to a pod's
+port. The supported implementation currently in OpenShift invokes `nsenter`
+directly on the node host to enter the pod's network namespace and invoke
```

Comment:
Yeah, couldn't decide, depending on how you read the sentence. Changed to "then invokes".